src/odemis/model/_metadata.py

```diff
@@ -184,7 +184,9 @@
 MD_FAV_POS_DEACTIVE = "Favourite position deactive"  # dict of str -> float representing a good position for being "deactive" (eg, mirror parked, lens not in use)
 MD_FAV_POS_COATING = "Favourite position coating"  # dict of str -> float representing a good position for GIS coating
 MD_FAV_AREA = "Favourite acquisition area"  # (float, float, float, float) minX, minY, maxX, maxY of the acquisition area
-MD_POS_ACTIVE_RANGE = "Range for active position"  # dict str → (float, float): axis name → (min,max): the range of the axes within which can be used imaging.
+MD_FAV_Z_RANGE = "Favourite acquisition range for Z axis"  # (float, float): (min,max): the range of the Z axis in milling position
```

Comment:
Added this metadata for advanced Z axis range, not sure if that's what you had in mind though.

Reply:
No need for FAV_Z_RANGE. We'll use POS_ACTIVE_RANGE.